stile/hsc/base_tasks.py

```diff
@@ -60,6 +60,7 @@ class CCDSingleEpochStileConfig(lsst.pex.config.Config):
     sys_tests = adapter_registry.makeField("tests to run", multi=True,
                     default=[#"StatsPSFFlux", #"GalaxyXGalaxyShear", "BrightStarShear",
                              "StarXGalaxyShear", "StarXStarShear", "Rho1",
+                             "StarSizeResidual",
```

Comment:
It's not clear that it is a correlation function from this name. Maybe `StarXStarSizeResidual` is better?